src/libraries/restore/runtime/referenceFromRuntime.targets

```diff
@@ -0,0 +1,12 @@
+<Project>
```

Comment:
Who is importing this?

Reply:
Does this need to be Directory.Build.targets?

Reply:
Actually its getting imported in restore\directory.build.targets for netcoreapp3.0-windowsNT.

Reply:
Can it be moved to Directory.Build.Targets for runtime.deproj and be imported always?

Reply:
I can try but I think importing it always will interfere with the outer build of the deprojs

Reply:
I see.  Now I understand.  Then directory.build.targets, import only when TargetFramework is set.  Add a comment that this is present in order to make ProjectReferences return the same set of content that would be BinPlaced.